### PR TITLE
Add framework specific rules

### PR DESCRIPTION
**Pull request by marcinwadon**
* Angular
* React
* Node

### CODE REVIEW

File: angular.tslint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["./tslint.json"],
+  "rules": {
+    "no-expression-statement": false,
+    "no-method-signature": false
+  }
+}
```
Comment (kpudlik) [inline angular.tslint.json]: newline

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "noUnusedLocals": true
+  },
+  "exclude": [
+    "node_modules"
+  ]
+}
```
Comment (kpudlik) [inline tsconfig.json]: newline

File: angular.tslint.json (hunk 1 above)
Comment (marcinwadon) [reply angular.tslint.json]: done

File: tsconfig.json (hunk 2 above)
Comment (marcinwadon) [reply tsconfig.json]: done

**Review by kpudlik — APPROVED** [review]

**Pull request merged**